.gitignore

```diff
@@ -3,3 +3,4 @@
 
 # PhpStorm Project Files
 .idea
+submissions/FabunmiTolulopeEniola/
```

Comment:
Why placing this file path in `.gitignore` file?

Reply:
I could not delete it